app/helpers/navigation_items.rb

```diff
@@ -77,7 +77,7 @@ def for_provider_primary_nav(current_provider_user, current_controller, performi
       if current_provider_user && !performing_setup
         items << NavigationItem.new('Applications', provider_interface_applications_path, active?(current_controller, %w[application_choices decisions offer_changes notes interviews offers feedback conditions reconfirm_deferred_offers]), [])
         items << NavigationItem.new('Interview schedule', provider_interface_interview_schedule_path, active?(current_controller, %w[interview_schedules]), [])
-        items << NavigationItem.new('Reports', provider_interface_reports_path, active?(current_controller, %w[reports application_data_export hesa_export recruitment_performance_reports]))
+        items << NavigationItem.new('Reports', provider_interface_reports_path, active?(current_controller, %w[reports application_data_export hesa_exports recruitment_performance_reports withdrawal_reports withdrawal_reasons_reports status_of_active_applications diversity_reports]))
```

Comment:
This fixes the navigation for all the reports -- it was only showing as highlighted for hesa and recruitment performance. Now it is highlighted for all the reports on the page.